project/src/components/favorites-card/favorites-card.tsx

```diff
@@ -0,0 +1,45 @@
+import {Link} from 'react-router-dom';
+import {Offer} from '../../types/offers';
+
+type OfferProps = {
+  offer: Offer;
+}
+
+function FavoritesCard(props: OfferProps): JSX.Element {
+  const {offer} = props;
+  return (
+    <article className="favorites__card place-card">
+      <div className="favorites__image-wrapper place-card__image-wrapper">
+        <Link to={`/offer/:${offer.id}`}>
+          <img className="place-card__image" src={offer.image[0]} width="150" height="110" alt="Place image"/>
```

Comment:
```suggestion
          <img 
            className="place-card__image"
            src={offer.image[0]} 
            width="150" 
            height="110" 
            alt="Place image"
          />
```

и заодно поправь линтинг

и src какой-то странный, пре называется image а там массив